CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+# NAME/CMakeLists.txt
+# Created by Robin Rowe 2020-05-01
+# License: License Public Domain
+
+# The process of building pForth involves several steps. 
+# cmake .. 
+# cd fth
+# ../build/Win32/Debug/pforth -i system.fth
+# Save the compiled dictionary as "pforth.dic".
+# The next time you run pforth, the precompiled pforth.dic file will be loaded automatically.
+
+cmake_minimum_required(VERSION 3.8)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+project(Pforth)
+message("Configuring ${PROJECT_NAME}...")
+enable_testing()
+if(WIN32)
+add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+message("Warning: _CRT_SECURE_NO_WARNINGS")
+endif(WIN32)
+add_subdirectory(csrc)
+link_libraries(${PROJECT_NAME}_lib)
+if(NOT WIN32 AND NOT APPLE)
+	link_libraries(rt pthread)
+endif(NOT WIN32 AND NOT APPLE)
+
+option(UNISTD "Enable libunistd" false)
+if(UNISTD)
+	set(LIBUNISTD_PATH /code/github/libunistd)
+	if(WIN32)
+		include_directories(${LIBUNISTD_PATH}/unistd)
+		link_directories(${LIBUNISTD_PATH}/build/unistd/Release)
+		link_libraries(libunistd)
+	endif(WIN32)
+endif(UNISTD)
+
+option(FLTK "Enable FLTK" false)
```

Comment:
What is FLTK?

Reply:
FLTK is a portable GUI lib. libunistd is a Windows POSIX library. Since both libs are set to option = false, neither is being used and may be deleted from cmake file as you wish.